app/views/clients/viewclient.html

```diff
@@ -965,11 +965,13 @@ <h4><strong>{{client.displayName}}</strong></h4>
 							<td>{{clientdocument.fileName}}</td>
 							<td>
 								<a target="_blank" href="{{hostUrl}}{{clientdocument.docUrl}}"><i class="fa fa-cloud-download"></i></a>&nbsp;
+								<a ng-click="previewDocument(clientdocument.docUrl,clientdocument.fileName)"><i class="fa fa-eye"></i></a>
```

Comment:
The icon should only be visible if the filetype is an Image.

Reply:
Ok, will add that.